kolibri/plugins/style_guide/assets/src/views/StyleGuideIndex.vue

```diff
@@ -20,6 +21,7 @@
 
   // Some components (e.g. <KNavbarLink>) need the router.
   FullVue.use(VueRouter);
+  FullVue.use(VueAphrodite);
```

Comment:
I actually have some major changes and fixes to the `style_guide` plugin coming up. Would you mind removing these changes to prevent conflict?